articles/azure-app-configuration/push-kv-devops-pipeline.md

```diff
@@ -103,3 +114,37 @@ To create Key Vault references, set the "Content Type" parameter to *application
 **Why am I receiving a 409 error when attempting to push key-values to my configuration store?**
 
 A 409 Conflict error message will occur if the task tries to remove or overwrite a key-value that is locked in the App Configuration store.
+
+**How can I create FeatureFlags using this task ?**
+
+To create feature flags, specify a **FeatureManagement** section in the configuration file and define the feature flags under the **FeatureManagement** section. The feature filters can be defined using the **EnabledFor** property. For the schema see [feature flag schema](https://github.com/microsoft/FeatureManagement-Dotnet/tree/main/docs/schemas). The following is an example of a json configuration file with feature flags defined.
+```json
+{
+    "FeatureManagement": {
+        "FeatureA": {
+            "EnabledFor": [
+                {
+                    "Name": "AlwaysOn"
+                }
+            ]
+        },
+        "FeatureB": {
+            "EnabledFor": [
+                {
+                    "Name": "TimeWindow",
+                    "Parameters": {
+                        "Start": "Wed, 23 November 2022 13:59:59 GMT",
+                        "End": "Mon, 05 December 2022 00:00:00 GMT"
+                    }
+                },
+                {
+                    "Name": "Percentage",
+                    "Parameters": {
+                        "PercentageFilterSetting": "50"
+                    }
+                }
+            ]
+        }
+    }
+}
+```
```

Comment:
For both feature flags and key vault references, I'm leaning towards redirecting customers to the [config file doc](https://learn.microsoft.com/en-us/azure/azure-app-configuration/concept-config-file). It has examples for both scenarios. It is also different for different content profiles.

Reply:
For key vault reference and feature flags should we still have a small explanation and then redirect them to [config file doc](https://learn.microsoft.com/en-us/azure/azure-app-configuration/concept-config-file)? Something like 

**How can I create Key Vault references using this task?**
To create Key Vault references, 
**Default FileContent Profile**: Set the "Content Type" task parameter to *application/vnd.microsoft.appconfig.keyvaultref+json;charset=utf-8*. If not all key-values in a configuration file are Key Vault references, put Key Vault references and normal key-values in separate configuration files, and push them separately. 
**KvSet FileContent Profile**: Set the configuration file "content_type" property value as *application/vnd.microsoft.appconfig.keyvaultref+json;charset=utf-8*.
 For more details see [Azure App Configuration support for configuration files](https://learn.microsoft.com/azure/azure-app-configuration/concept-config-file)

"**How can I create FeatureFlags using this task ?**

To create feature flags
**Default FileContent Profile**: Specify a **FeatureManagement** section in the configuration file and define the feature flags under the **FeatureManagement** section. The feature filters can be defined using the **EnabledFor** property. For the schema see [feature flag schema](https://github.com/microsoft/FeatureManagement-Dotnet/tree/main/docs/schemas). 
**KVSet FileContent Profile**: Set the configuration file "content_type" property value as *application/vnd.microsoft.appconfig.ff+json;charset=utf-8*.
For more details see [Azure App Configuration support for configuration files](https://learn.microsoft.com/azure/azure-app-configuration/concept-config-file).

or option 2

"**How can I create Key Vault references using this task?**"
See  [Azure App Configuration support for configuration files](https://learn.microsoft.com/azure/azure-app-configuration/concept-config-file)

"**How can I create FeatureFlags using this task ?**
See  [Azure App Configuration support for configuration files](https://learn.microsoft.com/azure/azure-app-configuration/concept-config-file)

Reply:
I'd like to avoid duplicated content. They are hard to maintain over time. I have a suggestion in your private repo PR.